CONTRIBUTING.md

```diff
@@ -20,10 +20,11 @@ Language packs are automatically generated once 95% of the plugin's strings are
 
 ### Testing translations in development without language packs
 
-1. Export the translations strings as a `.po` file from the [translate.wordpress.org project](https://translate.wordpress.org/projects/wp-plugins/woocommerce-admin).
-1. Rename `.po` file to the following format: `woocommerce-admin-xx_YY.po` (Where xx_YY is your locale, like it_IT)
-1. Use WP-CLI to generate needed JSON files for JavaScript-based strings: `wp i18n make-json [.po file here] --no-purge`
-1. Move `.po` and `.json` files to `/wp-content/languages/plugins`.
+1. Generate a translation file with `npm run i18n`.
+1. Rename `woocommerce-admin.pot` file to the following format: `woocommerce-admin-xx_YY.po` (Where xx_YY is your locale, like it_IT).
```

Comment:
Optional: if we're just using this to create a new language file, it would be a nice touch to allow an argument to be passed here to generate the file.  E.g., `npm run 18n --lang=it_IT`

Reply:
> it would be a nice touch to allow an argument to be passed here to generate the file.

Fantastic idea. When I updated the readme for this I did not note that is was optional because explaining both paths might make the instructions more confusing.